src/main/java/com/mcmoddev/alt/proxy/ServerProxy.java

```diff
@@ -4,19 +4,18 @@
 
 import net.minecraft.server.MinecraftServer;
 import net.minecraft.world.World;
-import net.minecraftforge.fml.server.FMLServerHandler;
 
 public class ServerProxy extends CommonProxy {
 	@Override
     public World getWorld() {
-		MinecraftServer mcServer = FMLServerHandler.instance().getServer();
-		
-		if( mcServer.worlds.length == 0 ) {
-			AdditionalLootTables.logger.warn("No World Ready Yet!");
+//		MinecraftServer mcServer = FMLServerHandler.instance().getServer();
+//		
+//		if( mcServer.worlds.length == 0 ) {
+//			AdditionalLootTables.logger.warn("No World Ready Yet!");
 			return null;

```

Comment:
Commands are handled only on the logical server side, so there should always be a MinecraftServer instance available. In the command handler I query its instance directly, so there's no need for proxies. Furthermore, loot tables are logical  server-side only, which means we should never worry about this.

Reply:
Ah, okay - in 1.12 it was also "logical server" but handled in the client while SMP handled it differently - and there was no easy access to the instance that I could find. This is a nice cleanup and I'm hoping it might be possible to back-port.